lightglue_onnx/superpoint.py

```diff
@@ -72,7 +72,7 @@ def max_pool(x):
 @torch.jit.script_if_tracing
 def top_k_keypoints(
     keypoints: torch.Tensor, scores: torch.Tensor, k: int
-) -> tuple[torch.Tensor, torch.Tensor]:
+):

```

Comment:
```suggestion
) -> Tuple[torch.Tensor, torch.Tensor]:

```

Reply:
Thanks for the suggestion. I added the correct type hint.